.github/workflows/ci.yml

```diff
@@ -16,7 +16,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [10, 12, 14, 16]
+        node-version: [14, 16]
```

Comment:
```suggestion
        node-version: [14, 16, 18]
```

Reply:
Node 18 is exploding due to some issues which seem related to older Got, and newer got explodes due to migration to ESM, and addressing this is beyond this particular PR

Reply:
I would love it if all Sindre modules were eradicated from any code base I am associated with.